src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -448,17 +466,28 @@ def get_webhook_data(
 def send_resource_change_webhook(
     installation_id: int, event: str, data: dict[str, Any], *args: Any, **kwargs: Any
 ) -> None:
-    installation = app_service.installation_by_id(id=installation_id)
-    if not installation:
-        logger.info(
-            "send_resource_change_webhook.missing_installation",
-            extra={"installation_id": installation_id, "event": event},
-        )
-        return
+    with SentryAppInteractionEvent(
+        operation_type=SentryAppInteractionType.SEND_WEBHOOK, event_type=event
+    ).capture() as lifecycle:
+        installation = app_service.installation_by_id(id=installation_id)
+        if not installation:
+            logger.info(
+                "send_resource_change_webhook.missing_installation",
+                extra={"installation_id": installation_id, "event": event},
+            )
+            return
 
-    send_webhooks(installation, event, data=data)
+        try:
+            send_webhooks(installation, event, data=data)
+        except SentryAppSentryError as e:
+            sentry_sdk.capture_exception(e)
+            lifecycle.record_failure(e)
+            return None
+        except (ApiHostError, ApiTimeoutError, RequestException, ClientError) as e:
+            lifecycle.record_halt(e)
+            raise
 
-    metrics.incr("resource_change.processed", sample_rate=1.0, tags={"change_event": event})
+        metrics.incr("resource_change.processed", sample_rate=1.0, tags={"change_event": event})
```

Comment:
will we remove these metrics once SLOs are finalized?

Reply:
yeah thats like the last milestone 🫠